.bazelci/config.yaml

```diff
@@ -48,6 +48,7 @@ tasks:
       - "-//cmake_defines/..."
       - "-//cmake_hello_world_lib/..."
       - "-//cmake_with_data/..."
+      - "-//cmake_with_target/..."
```

Comment:
If we defaulted to ninja then these would work on RBE...

Reply:
I'd rather spend next release getting RBE to work (build the make binary)